src/components/reusable/progressBar/progressBar.ts

```diff
@@ -0,0 +1,285 @@
+import { html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+import '../loaders';
+import '../tooltip';
+import checkmarkIcon from '@carbon/icons/es/checkmark--filled/16';
+import errorIcon from '@carbon/icons/es/error--filled/16';
+
+import ProgressBarStyles from './progressBar.scss';
+
+enum ProgressStatus {
+  ACTIVE = 'active',
+  SUCCESS = 'success',
+  ERROR = 'error',
+}
+
+/**
+ * `<kyn-progress-bar>` -- progress bar status indicator component.
+ * @slot unnamed - Slot for tooltip text content.
+ */
+@customElement('kyn-progress-bar')
+export class ProgressBar extends LitElement {
+  static override styles = ProgressBarStyles;
+
+  /** Sets visibility of optional inline load status spinner. */
+  @property({ type: Boolean })
+  showInlineLoadStatus = true;
+
+  /** Controls whether to show default helper text for active state. */
+  @property({ type: Boolean })
+  showActiveHelperText = true;
```

Comment:
Boolean props must default to false, ref: https://lit.dev/docs/components/properties/#boolean-attributes

Reply:
✅